inference-engine/src/inference_engine/include/openvino/runtime/remote_tensor.hpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a header file for the OpenVINO Runtime tensor API
+ *
+ * @file openvino/runtime/tensor.hpp
+ */
+#pragma once
+
+#include "common.hpp"
+#include "ie_remote_blob.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "tensor.hpp"
+
+namespace ov {
+namespace runtime {
+
+class RemoteContext;
```

Comment:
Can we avoid forward declaration?

Reply:
Not here. It avoids circular header dependency here.